module/entries/MassEdits/Edit-EntryPassword-All-Vaults.ps1

```diff
@@ -0,0 +1,64 @@
+#requires -version 4
+<#
+.SYNOPSIS
+  Change password on an individual Credential entry shared between all Vaults
+
+.DESCRIPTION
+  Option to change all Credential entries password sharing the same name across all Vaults
+  
+  SN: This is not a secure option we recommend having a seperate Vault with shared credentials 
+
+.PARAMETER <Parameter_Name>
+  None
+
+.INPUTS
+  Confirmation will be required for EACH Un-Parent
+
+.OUTPUTS
+  None
+
+.NOTES
+  Version:        1.0
+  Author:         Eric St-Martin
+  Creation Date:  05/05/2021
+  Purpose/Change: Initial script development
+.EXAMPLE
+
+#>
+
+#-------------------------------[Initialisations]------------------------------
+
+#Set Error Action to Silently Continue
+$ErrorActionPreference = 'SilentlyContinue'
+
+#Import Modules & Snap-ins
+Import-Module RemoteDesktopManager.PowerShellModule

```

Comment:
I would change to: 

if ( ! (Get-Module RemoteDesktopManager.PowerShellModule)) {
    Import-Module "${env:ProgramFiles(x86)}\Devolutions\Remote Desktop Manager\RemoteDesktopManager.PowerShellModule.psd1"
}